### ISSUE TITLE
FT LORA with InternVL2-26b: loss 0

### ISSUE THREAD

**Issue by alexemme** [modelscope/swift#1560]
Hello, I have a problem with the finetuning of InternVL2-26b. Initially, I used the original script provided on the project's GitHub page (`internvl2_26b_internlm2_20b_dynamic_res_2nd_finetune_lora.sh`), and the finetuning seemed to be successful, with the final training loss settling at a value close to 0. I merged the model with the LORA layers with the base model using swift, and it seemed to go smoothly. The fine-tuned model provides different responses from the base model.

I then tried to perform finetuning with swift following the instructions on the best practices page (https://swift.readthedocs.io/en/latest/Multi-Modal/internvl-best-practice.html) using the command:

CUDA_VISIBLE_DEVICES=0 swift sft --model_type internvl2-26b --model_id_or_path /pathto/InternVL2-26B --dataset /pathto/dataSwift.jsonl --max_length 4096 --use_flash_attn true --gradient_checkpointing true

The fine-tuning completes, but what I noticed that is not normal is that the indicated loss drops to zero shortly after starting, for example:

{'loss': 0.0, 'acc': 0.0, 'grad_norm': nan, 'learning_rate': 8.333e-05, 'memory(GiB)': 58.47, 'train_speed(iter/s)': 0.04858, 'epoch': 0.04, 'global_step/max_steps': '5/119', 'percentage': '4.20%', 'elapsed_time': '1m 41s', 'remaining_time': '38m 24s'}

I tried setting parameters similar to the official script that does not have problems:

--lora_lr_ratio 16 --learning_rate 2e-5 --weight_decay 0.05 --warmup_ratio 0.03

But the problem persists.

What could be the cause?
Could it be the dataset?
The dataset consists of about 2000 images with the same question and similar answers (I am testing the model for image classification).

**Comment by hjh0119** [comment]
It looks like the loss value overflowed. Will the subsequent training also be like this, or is it occasional? Maybe changing the random seed would help?

**Comment by alexemme** [comment]
After many hours of testing, I discovered that the problem was due to a faulty GPU. When fine-tuning the model using only the working GPU, the loss consistently remains below 0.5.
